pkg/apis/networking/validation/validation.go

```diff
@@ -21,8 +21,6 @@ import (
 	"net"
```

Comment:
don't change these files in this PR, limit this to the e2e / test changes

Reply:
:+1: